frontend/javascripts/oxalis/view/right-border-tabs/meshes_view.js

```diff
@@ -277,29 +278,43 @@ class MeshesView extends React.Component<Props, State> {
       }
     };
 
-    const getComputeMeshfileTooltip = node => (
-      <Tooltip
-        title={
-          features().jobsEnabled
-            ? "Compute a mesh file for this dataset so that meshes for segments can be loaded quickly afterwards."
-            : "The computation of mesh files is not supported by this webKnossos instance."
-        }
-      >
-        {node}
-      </Tooltip>
-    );
+    const getComputeMeshFileTooltipInfo = () => {
+      let title = "";
+      let disabled = true;
+      if (!features().jobsEnabled) {
+        title = "Computation jobs are not enabled for this webKnossos instance.";
+      } else if (this.props.hasVolume) {
+        title =
+          "Meshes cannot be computed for volume annotations. Please open this dataset in view mode to compute a mesh file";
```

Comment:
Why should `view` mode help here? `View` mode will not convert the volume annotation into a segmentation, right?

Also, if the toast supports HTML links, this should be a clickable link.

Reply:
> Why should view mode help here? View mode will not convert the volume annotation into a segmentation, right?

I changed the tooltip to `Meshes cannot be precomputed for volume annotations. However, you can open this dataset in view mode to precompute meshes for the dataset's segmentation layer.`. I hope this makes it clearer?

> Also, if the toast supports HTML links, this should be a clickable link.

It's a tooltip, so, no clickable link possible..

Reply:
This assumes that there is a segmentation layer to begin with in `view` mode. Depending on your dataset this is not the case. I am not sure if this already considered for this conditional. If so, than I guess its fine.

Reply:
Yeah, this case is already handled.